source/simulator/core.cc

```diff
@@ -1693,6 +1693,16 @@ namespace aspect
               solve_advection(AdvectionField::composition(c));
             }
 
+          if (
+            parameters.use_discontinuous_composition_discretization
+            &&
+            parameters.use_limiter_for_discontinuous_solution
```

Comment:
We should think now about if we need separate parameters for the composition and the temperature limiters. Do you think there could be a case where somebody only wants to apply the limiter to the composition field, but has a discontinuous temperate field as well? If yes, we need separate parameters for composition and temperature. I am asking because if we introduce this parameter, and apply it to all fields, it is cumbersome and incompatible to change its meaning to 'only apply to compositional fields' afterwards. What is your feeling about this?